docs/sources/dashboards/manage-dashboards/index.md

```diff
@@ -65,17 +67,17 @@ On the **Manage dashboards and folders** page, you can:
 
 ### Dashboard folder page
 
-You can complete the following tasks on the **Dashboard Folder** page:
+You can complete the following tasks on a dashboard folder page:
 
 - Move or delete dashboards in a folder
-- Rename a folder (available under the **Settings** tab)
+- Rename a folder (available on the Settings tab)
 - Assign permissions to folders (which are inherited by the dashboards in the folder)
 
-To navigate to the dashboard folder page, click the cog appears when you hover over a folder in the dashboard search result list or the **Manage dashboards and folders** page.
+To navigate to the dashboard folder page, hover over the name of the folder and click **Go to folder** in the dashboard search result list or on the Dashboards page.
```

Comment:
Doing a quick search in VS code, it seems like we're exclusively using "hover over". I'll bring it up with the team that we should encode that in our style guide since it doesn't seem like we're going to follow Google's lead on this one.